bears/general/AnnotationBear.py

```diff
@@ -12,16 +12,14 @@ class AnnotationBear(LocalBear):
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
 
-    def run(self, filename, file, language: str, coalang_dir: str = None):
+    def run(self, filename, file, language: str):
```

Comment:
keep the setting `coalang_dir` , but 'deprecate' it (see other deprecated settings in bears repo), and also dont attempt to re-implement it at all.

We might be able to re-implement this later, when coAST is usable.